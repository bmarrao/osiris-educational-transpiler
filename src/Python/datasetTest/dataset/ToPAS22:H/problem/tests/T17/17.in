5
12 0
xo xkEQ l oSYF Q m Xc UhGHd EG gN OahY uOY
12 10
x NmiOz UJnB QFrTl k o r immt Q a P X
Q QFrTl
UJnB k
o Q
immt UJnB
a r
immt x
P NmiOz
x P
NmiOz x
UJnB QFrTl
12 5
EsEBf xLQh BYfA Jr VT kPWX w WCoI M z k LVFk
LVFk EsEBf
w WCoI
LVFk Jr
EsEBf Jr
kPWX VT
12 6
q w ltBdS Uij Vh jQMOC FbMmJ RV xUDBj PM cNrZb SxFb
FbMmJ w
cNrZb w
Uij SxFb
q Uij
SxFb ltBdS
xUDBj ltBdS
12 3
lyEA z VXNpB l XnfG ScWxa pY SbT tx sJSO XAfI XnF
XAfI z
tx XnF
ScWxa XAfI
