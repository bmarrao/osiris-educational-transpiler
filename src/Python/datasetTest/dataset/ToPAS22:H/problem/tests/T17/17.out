Possivel: 479001600
Possivel: 178882560
Possivel: 293068800
Possivel: 266803200
Possivel: 358041600
